viper/parser/parser.py

```diff
@@ -225,22 +221,17 @@ def get_contracts_and_defs_and_globals(code):
 
 
 # Header code
-initializer_lll = LLLnode.from_list(['seq',
-                                        ['mstore', 28, ['calldataload', 0]],
-                                        ['mstore', ADDRSIZE_POS, 2**160],
-                                        ['mstore', MAXNUM_POS, 2**127 - 1],
-                                        ['mstore', MINNUM_POS, -2**127],
-                                        ['mstore', MAXDECIMAL_POS, (2**127 - 1) * DECIMAL_DIVISOR],
-                                        ['mstore', MINDECIMAL_POS, (-2**127) * DECIMAL_DIVISOR],
-                                    ], typ=None)
+initialiser_list = ['seq', ['mstore', 28, ['calldataload', 0]]]
+initialiser_list += [['mstore', pos, limit_size] for pos, limit_size in LOADED_LIMIT_MAP.items()]
+initializer_lll = LLLnode.from_list(initialiser_list, typ=None)
 
```

Comment:
I think this should be `initializer_list`.  Also it's easier for me to read if it's written like this:
```
initializer_lll = LLLnode.from_list(['seq', 
                                        ['mstore', 28, ['calldataload', 0],
                                        [['mstore', pos, limit_size] for pos, limit_size in LOADED_LIMIT_MAP.items()],
                                      ], typ=None)

Reply:
@DavidKnott sure! That was clearly my EN-GB kicking in :smile_cat:

Reply:
For the second part, unfortunately that would create a second sub-list, which isn't the same as the original sequence. I wasn't able to fine a more readable way to append the 'dynamic' LOADED_LIMIT_MAP list into the 'seq'.